osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt/src/main/resources/osgp-adapter-protocol-mqtt.properties

```diff
@@ -0,0 +1,129 @@
+
+#Paging
+paging.maximum.pagesize=30
+paging.default.pagesize=15
+
+# MQTT
+mqtt.broker.defaultPort=8883
+mqtt.broker.defaultTopics=+/measurement,+/congestion
+mqtt.broker.defaultQos=AT_LEAST_ONCE
+
+# =========================================================
+# MESSAGING CONFIG
+# =========================================================
+#----------------------------------------------------------
+# jms.default
+#----------------------------------------------------------
+#jms.default.broker.url=failover:(ssl://localhost:61617)
+jms.default.broker.url=failover:(tcp://localhost:61616)
+jms.default.connection.queue.prefetch=1
+
+#----------------------------------------------------------
+# jms.mqtt.requests
+#----------------------------------------------------------
+jms.mqtt.requests.queue=protocol-mqtt.1_0.osgp-core.1_0.requests
+jms.mqtt.requests.initial.redelivery.delay=300000
+jms.mqtt.requests.maximum.redelivery.delay=900000
+jms.mqtt.requests.redelivery.delay=300000
+
+#----------------------------------------------------------
+# jms.mqtt.responses
+#----------------------------------------------------------
+jms.mqtt.responses.queue=osgp-core.1_0.protocol-mqtt.1_0.responses
+jms.mqtt.responses.initial.redelivery.delay=300000
+jms.mqtt.responses.maximum.redelivery.delay=900000
+jms.mqtt.responses.redelivery.delay=300000
+
+#----------------------------------------------------------
+# jms.mqtt.log.item.requests
+#----------------------------------------------------------
+jms.mqtt.log.item.requests.queue=osgp.logging.protocol
+
+#----------------------------------------------------------
+# jms.osgp.requests
+#----------------------------------------------------------
+jms.osgp.requests.queue=osgp-core.1_0.protocol-mqtt.1_0.requests
+
+#----------------------------------------------------------
+# jms.osgp.responses
+#----------------------------------------------------------
+jms.osgp.responses.queue=protocol-mqtt.1_0.osgp-core.1_0.responses
+
+# =========================================================
+# Recover key settings
+# =========================================================
+executor.scheduled.poolsize=5
+key.recovery.delay=240000
+
+# =========================================================
+# Core retry settings
+#
+# Please note retries are scheduled as tasks and executed
+# by a ThreadPoolTaskScheduler. The effective minimum retry 
+# delay is therefore the interval of the scheduler.
+# =========================================================
+schedule.retry.mqtt.delay=30000
+schedule.retry.mqtt.retries=2
+
+schedule.retry.mqtt.backoff.multiplier=2
+schedule.retry.mqtt.backoff.exponential=true
+
+# device keys security
+encryption.key.path=/etc/osp/smartmetering/keys/platform/secret.aes
+
+encryption.rsa.private.key.path=/etc/osp/smartmetering/keys/platform/mqtt_device_keys_private
+encryption.rsa.public.key.path=/etc/osp/smartmetering/keys/platform/mqtt_device_keys_public
```

Comment:
Please clean up this property file. I assume these keys are not in use by the MQTT implementation.

Same for the properties below.

Reply:
Done